src/Database/LSMTree/Internal/MergingRun.hs

```diff
@@ -70,7 +70,7 @@ instance RefCounted m (MergingRun m h) where
     getRefCounter = mergeRefCounter
 
 data MergePolicyForLevel = LevelTiering | LevelLevelling
-  deriving stock (Show, Eq)
```

Comment:
I think you might have missed this suggestion: https://github.com/IntersectMBO/lsm-tree/pull/499#pullrequestreview-2505957129

Reply:
BTW, did we end up using these instances for the golden tests? It doesn't look like it -- if that's the case, let's remove these instances